db/reviews.cds

```diff
@@ -9,7 +9,7 @@ using my.bookshop.Books from './books';
 
 entity Reviews : cuid, managed {
     @cds.odata.ValueList
-    book     : Association to Books;
+    book     : Association to Books @mandatory @assert.target;
```

Comment:
Maybe move this down to line 19++ where also the other input validation annotations can be found

Reply:
Done.